packages/dds/tree/src/test/tsconfig.json

```diff
@@ -1,16 +1,13 @@
 {
-	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"extends": "../../../../../common/build/build-common/tsconfig.test.node16.json",
```

Comment:
why do some tsconfig's reference the base ones via packages and some via relative paths? We have ~227 path ones and ~58 package ones, so path seems to be more common: are we planning to move everything to the path based approach to avoid having to publish build-common to update them?

Reply:
I'm not sure. It seems like the path-based ones are used nearly universally for client packages: it looks like PropertyDDS packages might be the only exception. So perhaps it's because tooling was run on the client release group to auto-convert tsconfigs at some point, and then a few exceptions were made? Maybe @tylerbutler knows.

That said, I do like the path-based approach for the reason you mentioned.

Reply:
Yes we're trying to move to path-based config inheritance where possible.